var/spack/repos/builtin/packages/hdf5-vol-log/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hdf5VolLog(AutotoolsPackage):
+    """Log-based VOL - an HDF5 VOL Plugin that stores HDF5 datasets in a log-based
+    storage layout."""
+
+    homepage = 'https://github.com/DataLib-ECP/vol-log-based'
+    url      = 'https://github.com/DataLib-ECP/vol-log-based'
+    git = 'https://github.com/DataLib-ECP/vol-log-based.git'
+    maintainers = ['hyoklee']
+
+    version('master', branch='master', git=git,
+            commit='b13778efd9e0c79135a9d7352104985408078d45')
```

Comment:
Usually we just use:
```suggestion
    version('master', branch='master')
```
to target the master branch. You can also fix the commit `sha` but I don't think we should do both at the same time. Specifying `git` in the directive is not needed, since it's the same as the one at the package level.